src/test/java/com/karankumar/bookproject/backend/goal/CalculateReadingGoalTest.java

```diff
@@ -100,4 +101,11 @@ void testHowFarAheadOrBehindSchedule(){
         assertEquals(45, howFarAheadOrBehindSchedule(113,41));
         assertThat(howFarAheadOrBehindSchedule(113,86)).isZero();
     }
+
+    
```

Comment:
Could you please delete this line (the line above the `@Test` annotation)?

Reply:
Done.